pkg/version/version.go

```diff
@@ -48,7 +48,7 @@ var (
 	TimescaleVersionRangeString = struct {
 		Safe, Warn string
 	}{
-		Safe: ">=1.7.3 <2.1.0",
+		Safe: ">=1.7.3 <2.99.0",
```

Comment:
Why not do

```suggestion
		Safe: ">=1.7.3 <2.0.x",
```

as we are not sure if 2.1.x+ might bring some changes, which can be breaking.